src/app/containers/MostRead/index.jsx

```diff
@@ -16,6 +17,11 @@ const MostReadContainer = () => {
   const { variant } = useContext(RequestContext);
   const { service } = useContext(ServiceContext);
 
+  const { enabled } = useToggle('mostRead');
+  if (!enabled) {
+    return null;
+  }
+
```

Comment:
This container should render both the [AMP](https://github.com/bbc/simorgh/issues/4742) and Canonical container based on the `platform`, this work is currently in progress. I would like to retain the `if` block so that it doesn't interfere with that implementation.